library/ceph_volume.py

```diff
@@ -649,7 +649,8 @@ def run_module():
         try:
             report_result = json.loads(out)
         except ValueError:
-            strategy_change = "strategy changed" in out
+            strategy_change = ("strategy changed" in out or
+                               "strategy changed" in err)
             if strategy_change:
                 out = json.dumps(
                     {"changed": False, "stdout": out.rstrip("\r\n")})
```

Comment:
I didn't saw this the first time but we should also use err to print the output instead of out (`out.rstrip`) when the strategy changes ?

Reply:
Nice catch. I've added `err` too. However, it's redundant to make the user look at both stdout and stderr in case of a failure. So, I am printing those conditionally.

Thanks!